espnet2/tasks/asr.py

```diff
@@ -232,12 +247,6 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             default=None,
             help="The keyword arguments for joint network class.",
         )
-        group.add_argument(
-            "--model_conf",
```

Comment:
Sorry I cannot understand this part.
Why this option is removed?

Reply:
I've defined `model_choices` in L112 and added it to `class_choices_list` in L187. With this change, arguments from --model and --model_conf should be loaded by the `add_arguments` method in L330.

Reply:
Thanks. How is the help message (`help="The keyword arguments for model class."`) handled?

Reply:
It will be handled in https://github.com/espnet/espnet/blob/master/espnet2/train/class_choices.py#L94

Reply:
thanks!